RELEASE-NOTES.md

```diff
@@ -1,5 +1,15 @@
 # Prefect Release Notes
 
+## Release 2.13.4
+
```

Comment:
```suggestion
### Enhancements
- Allow task runs without a flow run — https://github.com/PrefectHQ/prefect/pull/10816
```

Reply:
There's a better way to say this

Reply:
I'm not totally sure this is worth calling out yet tbh - support for this is technically possible but not exactly friendly

Reply:
It would feel strange to me not to include this

Reply:
if users look to the release notes to upgrade, it makes sense for any possibly breaking changes to be there

Reply:
we could put it under the ###experimental heading..

Reply:
how about "Lay groundwork for allowing task runs without a flow run"?

Reply:
I don't think it warrants that - it's fine under the enhancements section